.sync.yml

```diff
@@ -16,9 +16,12 @@ appveyor.yml:
 Gemfile:
   required:
     ':system_tests':
-      - gem: beaker-hostgenerator
-        from_env: BEAKER_HOSTGENERATOR_VERSION
-      - gem: nokogiri
+      - gem: 'nokogiri'
```

Comment:
I thought the system tests section was going away?

Reply:
yeah  it should be removed

Reply:
Removed!